custom-routes-vis/src/main/webapp/script.js

```diff
@@ -73,6 +77,40 @@ function updateCoordinates(lat, lng, containerId) {
       "Latitude: " + lat + "<br>Longitude: " + lng + "<br>";
 }
 
+/**
+ * Updates the deep linking url to various test apps on the page.
+ * TODO: Update Route Token if we get access to Routes Preferred API.
+ */
+function updateDeepLinkingUrl() {
+  if (document.getElementById('origin-coordinates').innerHTML != '' && 
+      document.getElementById('destination-coordinates').innerHTML != '') {
+    document.getElementById(IOS_URL_ID).innerHTML =
+        '<a href=navsdkdemo://advanced?originLat=' + originDestinationMarkers[0].position.lat()
+        + '&originLng=' + originDestinationMarkers[0].position.lng() + '&destLat=' 
+        + originDestinationMarkers[1].position.lat() + '&destLng=' 
+        + originDestinationMarkers[1].position.lng() + '&routeToken=TOKEN>' 
+        + 'IOS Test App' + '</a>';
+    document.getElementById(V1_ANDROID_URL_ID).innerHTML =
+        '<a href=navsdk://fragmentactivity?originlat=' + originDestinationMarkers[0].position.lat() 
+        + '&originlng=' + originDestinationMarkers[0].position.lng() + '&destinationlat=' 
+        + originDestinationMarkers[1].position.lat() + '&destinationlng='
+        + originDestinationMarkers[1].position.lng() + '&precomputedroutetoken=TOKEN>' 
+        + 'Android V1 Test App' + '</a>';
+    document.getElementById(V2_ANDROID_URL_ID).innerHTML =
+        '<a href=navsdk://supportnavmapfragmentactivity?originlat=' 
+        + originDestinationMarkers[0].position.lat() + '&originlng='
+        + originDestinationMarkers[0].position.lng() + '&destinationlat=' 
+        + originDestinationMarkers[1].position.lat() + '&destinationlng='
+        + originDestinationMarkers[1].position.lng() + '&precomputedroutetoken=TOKEN>' 
+        + 'Android V2 Test App' + '</a>';
```

Comment:
Line break should come after operators.

https://google.github.io/styleguide/jsguide.html#formatting-where-to-break